webhooks-extension/docs/Limitations.md

```diff
@@ -0,0 +1,57 @@
+# Limitations
+<br/>
+
+- Only GitHub webhooks are currently supported.
+- Only `push` and `pull_request` events are currently supported, these are the events defined on the webhook.
+- A limited number of pipelines will currently work with this implementation (see below)
+
+## Pipeline Limitations
+
+It is worth noting that the webhooks extension does not currently work with all pipelines, it very specifically creates the following when the webhook is triggered:
+
+#### Git PipelineResource
+
+A PipelineResource of type git is created with:
+
+  - `revision` set to the short commit id from the webhook payload.
+  - `url` set to the repository url from the webhook payload.
+
+#### Image PipelineResource
+
+A PipelineResource of type image is created with:
+
+  - `url` set to ${REGISTRY}/${REPOSITORY-NAME}:${SHORT-COMMITID} where, REGISTRY is the value set when creating the webhook, other values are taken from the webhook payload.
```

Comment:
I'd add `code` markdown around the full registry bit, and then `REGISTRY` (also adds emphasis)

Reply:
changed to 

 - `url` set to `${REGISTRY}/${REPOSITORY-NAME}:${SHORT-COMMITID}` where, `REGISTRY` is the value set when creating the webhook, other values are taken from the webhook payload.